modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/sql/GridSqlStatement.java

```diff
@@ -67,4 +73,19 @@ public void limit(GridSqlAst limit) {
     public GridSqlAst limit() {
         return limit;
     }
+
+    /**
+     * @return Delimeter to use.
+     */
+    public static char delimeter() {
+        if (secureOutputToLog())
+            return SPACE_DELIM;
+
+        return DEFAULT_DELIM;
+    }
+
+    /** @return {@code True} if output is for log. */
+    public static boolean secureOutputToLog() {
+        return !GridSqlElement.INCLUDE_SENSITIVE && !GridSqlElement.INCLUDE_SENSITIVE_TL.get();
```

Comment:
Let's replace this with QueryUtils.includeSensitive() and hide all logic to utility class.

Reply:
Agreed, will fix.